vm/src/builtins.rs

```diff
@@ -536,6 +536,7 @@ fn builtin_pow(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
 }
 
 #[derive(Debug, FromArgs)]
+#[__inside_vm]
```

Comment:
What does this line do? Could you give it a more descriptive name?

Reply:
For now it's just to indicate that to refer to `rustpython_vm` we should use `crate::` instead of `::rustpython_vm::`. Once rust-lang/rust#57407 lands, we'll be able to get rid of it.